tasks.py

```diff
@@ -0,0 +1,91 @@
+import invoke
+from invoke import task
+import os
+from yaksh.settings import SERVER_POOL_PORT
+
+SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
+TARGET_CONTAINER_NAME = 'yaksh_code_server'
+SRC_IMAGE_NAME = 'fossee/yaksh_codeserver'
+
+def create_dir(path):
+    if not os.path.exists(path):
+        os.makedirs(path)
+
+@task
+def setupdb(ctx):
+    print("** Setting up & migrating database **")
+    ctx.run("python manage.py makemigrations")
+    ctx.run("python manage.py migrate")
+    ctx.run("python manage.py loaddata demo_fixtures.json")
+
+@task(setupdb)
+def serve(ctx):
+    print("** Running the Django web server. Press Ctrl-C to Exit **")
+    ctx.run("python manage.py runserver")
+
+@task
+def clean(ctx):
+    print("** Discarding database **")
+    ctx.run("rm -rf {0}".format(os.path.join(SCRIPT_DIR, 'db.sqlite3')))
+
+@task
+def getimage(ctx, image=SRC_IMAGE_NAME):
+    try:
+        result = ctx.run("sudo docker inspect {0}".format(image), hide=True)
+    except invoke.exceptions.Failure:
+        print("The docker image {0} does not exist locally".format(image))
+        print("\n** Pulling latest image <{0}> from docker hub **".format(image))
+        ctx.run("sudo docker pull {0}".format(image))
+
+@task
+def start(ctx, ports=SERVER_POOL_PORT, image=SRC_IMAGE_NAME, unsafe=False):
+    if unsafe:
+        with ctx.cd(SCRIPT_DIR):
+            print("** Initializing local code server **")
+            ctx.run("sudo python -m yaksh.code_server")
```

Comment:
This would run on python2. I am not sure if that is what we want

Reply:
I would not want to specify the version here, in case the user does not have python3 at all, the user is free to use a environment that defaults to Python3